portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -10,7 +10,7 @@
 public class FormHandlerServlet extends HttpServlet {
 
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {//POST request handler
```

Comment:
For methods, you usually want to put the comment above the function. In this case:

// some comment
@Override
public void ....

See this example for the industry standard for making Javadocs for documenting the purpose of a method:

https://www.tutorialspoint.com/java/java_documentation.htm#:~:text=version%20version%2Dtext-,Example,-Following%20program%20uses